src/network/netdev/bridge.c

```diff
@@ -160,6 +158,7 @@ static void bridge_init(NetDev *n) {
         b->mcast_snooping = -1;
         b->vlan_filtering = -1;
         b->stp = -1;
+        b->forward_delay = 15 * 1000000ULL;
```

Comment:
please use USEC_PER_SEC for this multiplication.

That said, I think we should use USEC_INFINITY as magic value here. Let's initialize this variable to that and then suppress setting the rtnl attrobute if it is still USEC_INFINITY.